requirements/requirements_git.txt

```diff
@@ -3,3 +3,4 @@ git+https://github.com/ansible/system-certifi.git@devel#egg=certifi
 git+https://github.com/ansible/ansible-runner.git@devel#egg=ansible-runner
 git+https://github.com/ansible/python3-saml.git@devel#egg=python3-saml
 django-ansible-base @ git+https://github.com/ansible/django-ansible-base@devel#egg=django-ansible-base[rest_filters,jwt_consumer,resource_registry,rbac]
+awx-plugins-core
```

Comment:
Is there a separate place for non-git deps, though?

Reply:
yeah, and it looks like our method for updating our requirements isn't quite working `Could not find a version that matches awx-plugins-core (from -r /awx_devel/requirements/requirements.in (line 5))`

Reply:
@chrismeyersfsu try replacing it with `awx-plugins-core >= 0.0.1a0` — perhaps this is because there's no stable releases, and it probably excludes pre-releases.

Reply:
same error.